services/app-api/forms/qm.ts

```diff
@@ -273,5 +309,5 @@ export const qmReportTemplate: ReportTemplate = {
       sidebar: false,
       elements: [],
     },
-  },
+  } as Record<MeasureTemplateName, MeasurePageTemplate>,
```

Comment:
oh is this how you fixed that really annoying ts error?! this is so much smarter than whatever i was trying to do 😂